snapcraft/__init__.py

```diff
@@ -446,3 +446,8 @@ def str_presenter(dumper, data):
 yaml.add_representer(str, str_presenter)
 yaml.add_representer(OrderedDict, dict_representer)
 yaml.add_constructor(_mapping_tag, dict_constructor)
+
+if common.is_snap():
```

Comment:
Ah! I couldn't remember why I didn't use "common" there. I was confused then, dug into the code to find common wasn't the common I'm used to, changed the code, and when you asked I couldn't see it from the code anymore. At least your ugly import is more obvious, haha.